plugins/org.eclipse.reddeer.swt/src/org/eclipse/reddeer/swt/impl/shell/OkCancelShell.java

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.reddeer.swt.impl.shell;
+
+import org.eclipse.reddeer.core.lookup.ShellLookup;
+import org.eclipse.reddeer.swt.impl.button.CancelButton;
+import org.eclipse.reddeer.swt.impl.button.OkButton;
+import org.eclipse.swt.widgets.Shell;
+import org.hamcrest.Matcher;
+
+/**
+ * Represents a simple shell with "OK" and "Cancel" buttons.
+ * 
+ * @author lvalach
+ *
+ */
+public class OkCancelShell extends DefaultShell {
+
+	/**
+	 * Instantiates a new OkCancelShell.
+	 *
+	 * @param text
+	 *            shell text
+	 */
+	public OkCancelShell(String text) {
+		super(ShellLookup.getInstance().getShell(text));
```

Comment:
you can call just super(text), the same applies for constructor with matchers, and also the empty one